WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -955,9 +958,12 @@ private extension ProductsViewController {
                                   onDataReloaded.merge(with: Just<Void>(())),
                                   // Giving it an initial value to enable the combined publisher from the beginning.
                                   onTableViewEditingEnd.merge(with: Just<Void>(())))
-            .sink { [weak self] selectedProduct, _, _ in
+            .withPrevious()
+            .sink { [weak self] previous, current in
```

Comment:
Nice suggestion! Since only the selected product is used, great idea to map the combined publisher to what's being used downstream. Updated in https://github.com/woocommerce/woocommerce-ios/pull/12060/commits/25d62b8c13f57d9c9664a63d9974525f462ef894